src/scss/styles.scss

```diff
@@ -3,6 +3,7 @@ All Styles
 **/
 
 //Global styles
+@import "slick-min";
```

Comment:
I wouldn't put this vendor stylesheet file at the very beginning of your imports. It should actually be grouped with your component stylesheets. I actually wrote about this source ordering in my latest yet-to-be-released Foundry post on CSS Architecture, but you can read about a good strategy for source ordering here:
https://www.xfive.co/blog/itcss-scalable-maintainable-css-architecture/